doc/transform_methods/linbasex.rst

```diff
@@ -0,0 +1,156 @@
+numerical effort for the interpretation of VM-images considerably. The obtained
+results produce directly the coefficients of the involved spherical functions, 
+making the reconstruction of sliced Newton spheres obsolete.*
+
+How it works
+------------
+
+.. figure:: https://cloud.githubusercontent.com/assets/10932229/14975430/ea9c25de-1144-11e6-8824-531c81976160.png
+   :width: 350px
+   :alt: projection
+   :align: right
+   :figclass: align-center
+
+   projections (Fig. 2 of [1])
+
+A projection of 3D Newton spheres along the :math:`x` axis yields a compact 1D function:
+
+.. math::
+
+ L(z, u) = \sum_k \sum_\ell P_\ell(u)P_\ell\left(\frac{z}{r_k}\right) \frac{\prod_{r_k}(z)}{2r_k} p_{\ell k}
+
+with :math:`u = \cos(\theta)`. This function constitutes a system of equations
+expressing :math:`L(z, u)` as a linear combination of :math:`P_\ell(z/r_k)`. There
+exists for a given base a unique set of coefficients :math:`p_{\ell k}` 
+producing a least-squares fit to the function :math:`L(z, u)`.
+
+|
+|
+
+
+
+[ *extract of a comment made by Thomas Gerber (method author)* ]
+
+*Imaging an PES experiment which produces electrons that are distributed on the 
+surface of a sphere. This sphere can be described by spherical functions. If 
+all electrons have the same energy we expect them on a (Newton) sphere with 
+radius* :math:`i`. *This radius is projected to the CCD. The distribution on 
+the CCD has (if optics are approriate) the same radius* :math:`i`. 
+*Now let us assume that the distribution on the Newton sphere has some 
+anisotropy. We can describe the 
+distribution on this sphere by spherical functions* :math:`Y_{nm}`. 
+*Let's say* :math:`xY_{00} + yY_{20}`. 
+*The 1D projection of those spheres produces just* :math:`xP_{i0}(k) +yP_{i2}(k)`
+*where* :math:`P_{i}` *denotes Legendre Polynomials scaled to the interval* 
+:math:`i` *and* :math:`k` *is the argument (pixel).*
+
+*For one projection Lin-Basex now solves for the parameters* :math:`x` *and* 
+:math:`y`. *If we look at another projection turned by an angle, the Basis* 
+:math:`P_{i0}` *and* :math:`P_{i2}` 
+*has to be modified because the projection of e.g.,* :math:`Y_{20}` *turned 
+by an angle yields another function. It was shown that this function for e.g.,* 
+:math:`P_{2}` *is just* 
+:math:`P_{2}(a)P_{i2}(k)` *where* :math:`a` *is the turning angle. Solving 
+the equations for the 1D projection at angle* (:math:`a`) *with this modified 
+basis yields the same* :math:`x` and :math:`y` *parameters as before.*
+
+*Lin-Basex aims at the determination of contributions in terms of spherical 
+functions calculating the weight of each* :math:`Y_{l0}`. *If we reconstruct 
+the 3D object by adding all the* :math:`Y_{l0}` *contributions we get the 
+inverse Laplace transform of the image on the CCD from which we can derive 
+"Slices".*
+
+
+When to use it
+--------------
+[ *another extract from comments by the method author Thomas Gerber* ]
```

Comment:
I would probably paraphrase Thomas's comments, since this allows us to improve them in the future as our understanding improves. With a quote, you are locked in forever.